### DISCUSSION TITLE
so yeah

### DISCUSSION THREAD

**Discussion by TodePond** [TodePond/GulfOfMexico#16] [General]
um yeah so

**Comment by TodePond** [comment]
great chat

**Reply by stohrendorf** [reply]
how's the weather at your location?

**Comment by ghost** [comment]
I think this is the general general chat
Also, your language is blowing up on the Internet!
IT'S WORKING!
Every company shall use our language!!!
```javascript
const const const company = DREAMBERD!!!!!!!!!!!!!!!!!
```

https://www.youtube.com/shorts/-tCi-SX8vw8

**Comment by GitisAwesome** [comment]
Well hello everybody.

**Comment by GitisAwesome** [comment]
Why don't we all work together make a compiler or interpreter for this language?

**Reply by stohrendorf** [reply]
See #183

**Reply by GitisAwesome** [reply]
They all haven't been updated for 2 months tho.

**Reply by GitisAwesome** [reply]
Hello TodePond

**Reply by GitisAwesome** [reply]
I can't really write a programming language but I do know how it works.

**Reply by GitisAwesome** [reply]
Probably have to learn C for that.
Yes I am going to write C in C if I can.
